src/coreclr/src/pal/inc/pal.h

```diff
@@ -1108,7 +1099,6 @@ OpenSemaphoreW(
     IN LPCWSTR lpName);
 
 #ifdef UNICODE
-#define CreateSemaphore CreateSemaphoreW
 #define CreateSemaphoreEx CreateSemaphoreExW
 #else
 #define CreateSemaphore CreateSemaphoreA
```

Comment:
I am wondering whether anything uses the `A` versions of the `CreateSemapthore`, `CreateEvent`, etc. synchronization methods. If not, it should be fine to delete them.

Reply:
Deleted `CreateSemaphoreA`, but `CreateEvent{A,W}` etc. are currently used in debugger and toolbox so I have not touched those.

Reply:
I meant to delete all `*A` flavors of these APIs: CreateEventA, CreateSemaphoreA, CreateSemaphoreExA, CreateMutexA. I  think none of these should be used anymore (and if they are - it should be easy to flip the callsite to use the `W`  flavor).

Reply:
They are currently used in debugger and toolbox, where on Windows it may fail with W flavor. I didn't touched those, because that debugger/toolbox code was way out of PAL. Can give it a try.

Reply:
I think it would be useful to give it a try.

Reply:
Removed. At least tests are happy on my macOS. Lets see how it goes in the CI.